solutions/fully-configurable/provider.tf

```diff
@@ -7,3 +7,13 @@ provider "ibm" {
   region           = var.region
   visibility       = var.provider_visibility
 }
+
+# Null resource replaced with restapi_object to enable CSPM
+provider "restapi" {
+  uri = "https://resource-controller.cloud.ibm.com"
```

Comment:
We should be hitting the private endpoint

Reply:
Hmm, we also need to consider how we would honour environment variables that users may use to override defaults so it can deploy to test.cloud.ibm.com